### DISCUSSION TITLE
XTouch Compact

### DISCUSSION THREAD

**Discussion by Obandi** [bjoluc/cubase-mcu-midiremote#19] [Ideas]
Just wondering if you can consider making a config for Xtouch compact device. Thanks.

**Comment by bjoluc** [comment]
Hi @Obandi,
thanks for getting in touch! With its 16 push encoders, the X-Touch Compact is a bit of an exception among MCU-compliant devices and the hardware is a bit wasted in MCU mode (with the right 8 encoders only working as buttons AFAIK), so I think the X-Touch Compact would be best used with a custom non-MCU script. Unfortunately, that's nothing I intend to do – sorry! I guess it's only a matter of time though until someone builds a proper script.
